src/Vlingo.Directory/Model/Properties.cs

```diff
@@ -6,34 +6,29 @@
 // one at https://mozilla.org/MPL/2.0/.
 
 using System;
-using System.Collections.Generic;
 using System.IO;
-using System.Linq;
+using Vlingo.Common;
 
 namespace Vlingo.Directory.Model
 {
-    public sealed class Properties
+    public sealed class Properties : ConfigurationProperties
     {
-        private static readonly string _propertiesFile = "vlingo-cluster.properties";
+        private static readonly string _propertiesFile = "vlingo-directory.json";
 
-        private static Func<Properties> _factory = () => Open();
+        private static Func<Properties> Factory = () =>
```

Comment:
Why changing the name from `_factory` ? keep the same like it was

Reply:
renamed